### DISCUSSION TITLE
NamedCommands Help!

### DISCUSSION THREAD

**Discussion by NCJazzMan65** [mjansen4857/pathplanner#637] [General]
I'm the lead programming mentor for FRC 4121.  We are integrating PathPlanner into our Java robot code.  We are trying to use NamedCommands to have some of our commands (i.e. RunIntake, RunShooter, etc.) execute during our paths.  We've imported NamedCommands into RobotContainer, but when we try to create a NamedCommand, VSCode doesn't recognize the registerCommand method.  In fact, it doesn't recognize any methods of the NamedCommands class.  We need some help, please.

**Comment by WarrenReynolds** [comment]
Do you have the following line in your robotcontainer where the named commands should live?
```import com.pathplanner.lib.auto.NamedCommands;```
